cantor-phoenix/src/main/java/com/salesforce/cantor/phoenix/PhoenixDataSourceProvider.java

```diff
@@ -0,0 +1,60 @@
+package com.salesforce.cantor.phoenix;
+
+import com.zaxxer.hikari.HikariDataSource;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import javax.sql.DataSource;
+
+public class PhoenixDataSourceProvider {
+    private static final Logger logger = LoggerFactory.getLogger(PhoenixDataSourceProvider.class);
+
+    public static synchronized DataSource getDatasource(final PhoenixDataSourceProperties builder) {
+        return doGetDataSource();
+    }
+
+    private static DataSource doGetDataSource() {
+        final String jdbcUrl = "jdbc:phoenix:akouthoofd-wsl";
+        try {
+            // force loading Phoenix drivers
+            Class.forName("org.apache.phoenix.jdbc.PhoenixDriver");
+        } catch (ClassNotFoundException e) {
+            throw new RuntimeException(e);
+        }
+
+        final HikariDataSource connectionPoolDataSource = new HikariDataSource();
+
+        connectionPoolDataSource.setJdbcUrl(jdbcUrl);
+
+        connectionPoolDataSource.setConnectionTimeout(30 * 1000);  // 30 seconds connection timeout
+        connectionPoolDataSource.addDataSourceProperty("useUnicode", true);
```

Comment:
Are these parameters actually valid for the phoenix client?

Reply:
These parameters work with phoenix client, but I do not think they are needed. I removed all of them in the newest commit. It can be added back as needed when working with production data.